File: BTT4_PY1/BTT4_PY1/AddFriendGUI.py
from Controller import Controller
class AddFriendGUI(object):
	def RequestToAddFriend1(userID):
		contr = Controller
		print("input friend's userName: ")
		userNameFriend= input()
		print("userNameFriend: ",userNameFriend)
		contr.RequestToAddFriend(userID, contr.TakeUserID(userNameFriend))

File: BTT4/BTT4 (BTT3 UPDATE)/BTT3/AddFriendGUI.cpp
#include "AddFriendGUI.h"
#include "Controller.h"

void AddFriendGUI::InputFriendUserName(string userID)
{
	Controller Controller_GUI;
	cout << "input userName of friend" << endl;
	string friendName;
	cin >> friendName;
	Controller_GUI.RequestAddFriend(userID, friendName);
}

AddFriendGUI::AddFriendGUI()
{
}
AddFriendGUI::~AddFriendGUI()
{
}
